update readme (#1)
DevExpress-Examples/winforms-grid-multiselect-rows-with-checkbox-column: Readme.md

[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
<!-- default badges end -->

> # Data Grid for Windows Forms - How to use an unbound check box column to select grid rows

Author: ```suggestion
# Data Grid for Windows Forms - How to use an unbound checkbox column to select grid rows
```

> The [Grid View](https://docs.devexpress.com/WindowsForms/3464/controls-and-libraries/data-grid/views/grid-view) in the [Data Grid](https://docs.devexpress.com/WindowsForms/3455/controls-and-libraries/data-grid) control supports a built-in checkbox column for multiple row selection. 

Reviewer: On line 7 above you mention "check box", but on this line you say "checkbox". Can you please make it consistent?

> Set the following properties as follows to enable this checkbox column:

Reviewer: ```suggestion
Set the properties below as follows to enable this checkbox column:
```

- [View.OptionsSelection.MultiSelect](https://docs.devexpress.com/WindowsForms/DevExpress.XtraGrid.Views.Base.ColumnViewOptionsSelection.MultiSelect) property to `true`.
- [View.OptionsSelection.MultiSelectMode](https://docs.devexpress.com/WindowsForms/DevExpress.XtraGrid.Views.Grid.GridOptionsSelection.MultiSelectMode) property to `CheckBoxRowSelect`.

> In v18.1 and later versions, you can sync row selected states with a Boolean field in the bound data source. A checkbox column's bound mode ensures that row selection persists whenever you filter or group grid data. See the following topic for more information: [Multiple Row Selection via Built-In Check Column and Selection Binding](https://docs.devexpress.com/WindowsForms/16439/controls-and-libraries/data-grid/focus-and-selection-handling/multiple-row-selection-via-built-in-check-column-and-selection-binding)

Reviewer: ```suggestion
In v18.1 and later versions, you can sync row selected states with a Boolean field in the bound data source. A checkbox column's bound mode ensures that row selection persists whenever you filter or group grid data. See the following topic for more information: [Multiple Row Selection Using Built-In Check Column and Selection Binding](https://docs.devexpress.com/WindowsForms/16439/controls-and-libraries/data-grid/focus-and-selection-handling/multiple-row-selection-via-built-in-check-column-and-selection-binding).
```

Reviewer: Can you please update the actual topic title as well?

## Documentation

> - [Multiple Row Selection via Built-In Check Column and Selection Binding](https://docs.devexpress.com/WindowsForms/16439/controls-and-libraries/data-grid/focus-and-selection-handling/multiple-row-selection-via-built-in-check-column-and-selection-binding)

Reviewer: ```suggestion
- [Multiple Row Selection Using Built-In Check Column and Selection Binding](https://docs.devexpress.com/WindowsForms/16439/controls-and-libraries/data-grid/focus-and-selection-handling/multiple-row-selection-via-built-in-check-column-and-selection-binding)
```
